workers/social/lib/social/models/computeproviders/stacktemplate.coffee

```diff
@@ -435,7 +435,14 @@ module.exports = class JStackTemplate extends Module
 
             data['meta.modifiedAt'] = new Date
 
-            next()
+            originalId = data.config?.clonedFrom
+            return next()  unless originalId
+
+            # update clonedSum information in the template, if it is exists
+            JStackTemplate.one$ client, { _id: originalId }, (err, template) ->
+              if template and not err
+                data.config.clonedSum = template.template.sum
```

Comment:
hmmm you update stack B while ignoring the update in stack A. Yes you are right, editor will think that I have same content with original one but it doesn't.
Whether I am updating the cloned stack template with original one or changes that I made, the config of clonedSum should be sync with original one. It only affects to show or not the warning view
I think the code should work with this scenario as well. I will give a try soon and update you.